enlighten/file_io/Configuration.py

```diff
@@ -341,11 +341,15 @@ def has_section(self, section):
             return section in self.defaults
 
     def has_option(self, section, option):
+        # any option which is defaulted will always indicate "has_option -> true";
+        # .ini file may override value, but can't change that there IS a value
+        if section in self.defaults and option in self.defaults[section]:
+            return True
```

Comment:
This was preventing "CSV" from being checked by default on new installations (no enlighten.ini file)